packages/snaps-sdk/src/jsx/components/Text.ts

```diff
@@ -19,6 +30,7 @@ export type TextChildren = SnapsChildren<
 export type TextProps = {
   children: TextChildren;
   alignment?: 'start' | 'center' | 'end' | undefined;
+  color?: TextColors;
```

Comment:
```suggestion
  color?: TextColors | undefined;
```

Reply:
Isn't that redundant? I thought with the `?` operator it means the same thing?

Reply:
It does, but when using `exactOptionalProperties` the behavior is different, so as a minor dev-ex thing we do it this way.

https://www.typescriptlang.org/play/?ts=4.4.2&exactOptionalPropertyTypes=true#example/exact-optional-properties